<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTextTableDialog</class>
 <widget class="QDialog" name="CreateTextTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ширина отступов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="indentSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Количество строк</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rowsSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Количество столбцов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="columnsSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="createTable">
     <property name="text">
      <string>Создать таблицу</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
